### PR TITLE
Azure NextGen tutorials

### PR DESCRIPTION
**Pull request by mikhailshilkov**
- Support Azure NextGen in tutorial generation
- Add a landing page for Azure NextGen tutorials
- Regenerate all tutorials

Part of https://github.com/pulumi/docs/issues/5185 (more to come when we have more examples)

### CODE REVIEW

**Comment by cnunciato** [comment]
Should we add a top-level item at [/docs/tutorials](https://pulumi.com/docs/tutorials) for these as well?

File: content/docs/tutorials/azure-nextgen/_index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        <p>
+            Provision a Linux web server VM with a public IP address.
+        </p>
+        <p>
+            <a href="{{< relref "azure-nextgen-ts-webserver" >}}" style="color: #4387c7">
+                TypeScript
+            </a>&bull;
+            <a href="{{< relref "azure-nextgen-py-webserver" >}}" style="color: #4387c7">
+                Python
+            </a>&bull;
```
Comment (cnunciato) [inline content/docs/tutorials/azure-nextgen/_index.md:117]:
```suggestion
            </a>
```

**Review by cnunciato — APPROVED** [review]
Couple of minor questions about the ordering of these in the left nav and the replacement of the Azure tutorials on the index page, both of which I'll defer to @davidwrede on. Otherwise, just one minor nit. 👍

**Comment by mikhailshilkov** [comment]
@cnunciato Great points! I won't merge this until we are final on naming and positioning the two providers against each other.

**Comment by mikhailshilkov** [comment]
Shipped in another PR

**Pull request closed without merging**